projects/1.18-fabric/assets/exlines-sushi-mod/sushimod/lang/zh_cn.json

```
@@ -1 +1,23 @@
-{}
\ No newline at end of file
+{
+  "itemGroup.sushimod.foods": "寿司和刺身",
+  "item.sushimod.cucumber": "黄瓜",
+  "item.sushimod.cucumber_seeds": "黄瓜种子",
+  "item.sushimod.calamari_roll": "鱿鱼卷",
+  "item.sushimod.calamari_sashimi": "鱿鱼刺身",
+  "item.sushimod.clownfish_roll": "小丑鱼卷",
+  "item.sushimod.clownfish_sashimi": "小丑鱼刺身",
```

```suggestion
  "item.sushimod.clownfish_sashimi": "尼莫刺身",
```
本来我想吐槽不都是Clownfish为何一个小丑鱼一个尼莫（鱼）。
看了原文是`Nemo Sashimi`确信是海底总动员梗（什么魔鬼）。
所以这个`尼莫`我觉得应该保留，且不用加“鱼”字，作者想表达的就是“这是用`尼莫`做的刺身”。